src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -126,7 +125,6 @@ private void initializeInfrastructure() {
         networkState.registerHandler(context.get(CISShadowMonitor.class));
         networkState.registerHandler(context.get(BackgroundCertificateRefresh.class));
         context.get(BackgroundCertificateRefresh.class).start();
-        context.get(MetricsEmitter.class).start(MetricsConfiguration.DEFAULT_PERIODIC_AGGREGATE_INTERVAL_SEC);
```

Comment:
how is the emitter being started now?

Reply:
When auth service subscribes to config changes it reads in the metrics config and starts based off of what's specified in the config. Starting it here was actually redundant since it would restart after the subscribeToConfigChanges anyways